src/Elasticsearch.Net/Responses/ServerError.cs

```diff
@@ -115,6 +115,7 @@ internal static Error Create(IDictionary<string, object> dict, IJsonSerializerSt
 			: $"Type: {this.Type} Reason: \"{this.Reason}\" CausedBy: \"{this.CausedBy}\"";
 	}
 
+	[Obsolete("Removed in 6.0.")]
 	public class CausedBy
```

Comment:
While technically true this is only used through reading server errors, when upgrading no code for the property access into this needs to be changed.

Reply:
Rather leave this out to reduce noise.